### DISCUSSION TITLE
features shape

### DISCUSSION THREAD

**Discussion by dossos** [fraunhoferportugal/tsfel#139] [General]
cfg_file has three dictionaries spectral, statistical, and temporal and their total defined features are 60, but why do the generated feature matrix columns usually over 60 and get over 100? I understand the feature matrix row is decided by input data divided by window_size, who decided the columns?

**Comment by dmfolgado** [comment]
Hi! Some features, like the histogram, FFT, and Wavelet, produce multiple values that are distributed across different columns. To determine the total number of features (columns) based on a specific configuration dictionary, you can use the `tsfel.get_number_features method`.
